cmake/plugin-config.cmake

```diff
@@ -0,0 +1,10 @@
+# Find VTK at top level as f3d_vtkext-targets need it
+find_package(VTK 9.0 REQUIRED COMPONENTS CommonCore CommonExecutionModel IOImport)
+include("${CMAKE_CURRENT_LIST_DIR}/f3dPlugin.cmake")
+include("${CMAKE_CURRENT_LIST_DIR}/../f3d_vtkext/f3d_vtkext-targets.cmake")
```

Comment:
can't we add `f3d_vtkext-targets.cmake` to the current directory?

Reply:
no, this is a limitation of the vtk_module_build macro. it basically install the targets file to `${package}/${package}-targets.cmake`. This is something that could be changed but in any case, we will need compatibility with older VTK anyway.

Once this is merged, I will see If I can improve this in VTK.

Reply:
too bad :(